invoice/models.py

```diff
@@ -149,12 +149,12 @@ class InvoiceItem(Model):
     @property
     def net_total(self) -> Decimal:
         """Get the sum of the item excluding taxes."""
-        return self.price * self.quantity
+        return round(self.price * self.quantity, 2)
```

Comment:
does this rounding even work?
I think you need to use the decimal methods

Reply:
Actually it does. 
However, I would prefer to do it on the views/PDFs, but haven't figured out on how to do it in the PDF creation .

Reply:
I tested `round` and it works.
Explicitly rounding is bad though, we want to use a lot of precision during the calculations and only do rounding at the very end.
And we should probably use Decimal contexts

Reply:
Yes, in theory we would never be required to round.  Unless we have fractional quantities or very small prices. So, it is always on item level and does not affected invoices totals.